partiql-ast/src/main/resources/partiql_ast.ion

```diff
@@ -829,6 +837,9 @@ table_definition::{
       name:        '.identifier.symbol',
       type:        '.type',
       constraints: list::[constraint],
+      // Should this be a constraint?
+      is_optional: bool,
```

Comment:
Maybe we can prepend this with a `TODO` to make discoverable later. Also, on syntax, do this matter to be a constraint or not?

Reply:
I don't think it matters on the syntax. 

The only thing is that Attribute Constraint in SQL is often after the data type declaration, but this is more of a terminology things.